src/main/kotlin/no/nav/klage/clients/JoarkClient.kt

```diff
@@ -123,4 +123,15 @@ class JoarkClient(
         val journalpostCopyWithoutFileData = journalpost.copy(dokumenter = dokumenterWithoutFileData)
         return jacksonObjectMapper().writeValueAsString(journalpostCopyWithoutFileData)
     }
+
+    private fun getBehandlingstema(klage: Klage): String {
+        return if (klage.isLonnskompensasjon())
+            BEHANDLINGSTEMA_LONNSKOMPENSASJON
+        else
+            BEHANDLINGSTEMA_KLAGE_UNDERINSTANS
+    }
+
+    private fun Klage.isLonnskompensasjon(): Boolean {
+        return tema == "DAG" && ytelse == "Lønnskompensasjon for permitterte"
```

Comment:
Hvor kommer ytelse fra her? Er det det som er sendt inn som URL-parameter? Er i så fall denne sjekken trygg nok? Ser at det er norske bokstaver i navnet. Kan vi risikere at noe URL-encoding i visse browsere kan ødelegge dette?

Reply:
Skjønner at du spør. Denne blir satt i klage-dittnav-api, basert på en nøkkel sendt inn fra klienten her: https://github.com/navikt/klage-dittnav-api/blob/5e4b34572a387065ad3c261fe901feb5da385fd6/src/main/kotlin/no/nav/klage/domain/titles/TitleEnum.kt#L109

Så denne strengen er ikke innom browser, men blir sendt fra klage-dittnav-api til denne appen. Fra før har vi bare brukt `ytelse` som input til pdf-genereringen, så kom dette unntaket for akkurat denne ytelsen.